cpp/spdlog/spdlog.go

```diff
@@ -0,0 +1,35 @@
+package spdlog
+
+import (
+	_ "unsafe"
+
+	"github.com/goplus/llgo/c"
+)
+
+const (
+	LLGoPackage = "link: $(pkg-config --libs cppWrap spdlog); -lcppWrap -lspdlog -pthread -lfmt"
+)
+
+//go:linkname PrintInfo C.PrintInfo
+func PrintInfo(msg *c.Char)
```

Comment:
should have namespace: eg. `spdlogPrintInfo`

Reply:
ok, the func name in code has change to spdlogFunc

Reply:
when added nameSpace, the qiniu-x remind:
![error](https://github.com/user-attachments/assets/6251073b-e594-4c7f-8bd4-4c654ad5d410)

have removed namespace